### PR TITLE
ZH-trans: add kubeadm_alpha_phase_certs_front-proxy-ca.md

### PR DESCRIPTION
**Pull request by xichengliudui**
ZH-trans: add kubeadm_alpha_phase_certs_front-proxy-ca.md

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_front-proxy-ca.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+<!--
+If both files already exist, kubeadm skips the generation step and existing files will be used. 
+-->
+如果两个文件都已存在，kubeadm 将跳过生成步骤，并将使用现有文件。
+
+<!--
+Alpha Disclaimer: this command is currently alpha.
+-->
+Alpha 免责声明：此命令目前属于 alpha。
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_front-proxy-ca.md]:
```suggestion
 Alpha 免责声明：此命令目前属于 alpha 阶段。
```

**Comment by Hartmarken348** [comment]
11967

**Pull request merged**